packages/reactDom/src/ReactDOM.ml

```diff
@@ -18,6 +18,11 @@ let attribute_to_html attr =
   (* true attributes render solely the attribute name *)
   | Bool (name, _, true) -> Html.present name
   | Style styles -> Html.attribute "style" (ReactDOMStyle.to_string styles)
+  (*
+     When we have a Action prop, means that we have a `server action` and not a `string action` in this prop, them we don't render it
+     because the server action will be provided as a model and react will handle the rest.
+  *)
+  | Action (_name, _, _) -> Html.omitted ()
```

Comment:
nit: can you make this comment like 1 line and 5 words?

```
server actions get ignored blabla
```

Reply:
also, I wonder if we need to raise when seeing one Action passed in ReactDOM, since actions in HTML do not make any sense